pageStyles/candidates.module.scss

```diff
@@ -0,0 +1,43 @@
+﻿@import 'pageStyles/constants';
+
+.link {
+  margin: unset;
+}
+
+.desktopButton {
+  background-color: $softwireYellow;
+  border: none;
+  padding: 5px 15px;
+  font-size: 20px;
+  font-weight: bold;
+  margin-top: 10px;
+  position: fixed;
+  display: none;
+  
+  :hover{
+    text-decoration: underline;
+    cursor: pointer;
+  }
+    
+  @media (min-width: 768px) {
+    right: 10px;
+    top: 265px;
+    display: block;
+  }
+}
+
+.mobileButton {
+  background-color: $softwireYellow;
+  padding: 1px 8px 0 7px;
```

Comment:
this looks odd too.. I'd expect it to be symmetrical?

Reply:
sorted this too